CHANGELOG.md

```diff
@@ -2,7 +2,7 @@ CHANGELOG
 =========
 
 ## HEAD (Unreleased)
-(None)
+- Retain `id` fields in output types and resource properties [#521](https://github.com/pulumi/pulumi-google-native/pull/521)
```

Comment:
Do you want to highlight this as a breaking change for a number of resource input args?